docs/use/Set-Up-DeFi-Tools/index.md

```diff
@@ -0,0 +1,62 @@
+---
+id: set-up-defi-tools
+order: 12
+title: Set up DeFi Tools
+---
+
+## Setting up Metamask
+### What is Metamask?
+MetaMask is a browser extension that allows you to use Ethereum Protocols or `dApps` ("decentralized Apps") such as Tinlake right in your browser. Metamask holds your private keys, and once it's set up, you can connect it to the dApp in your browser and confirm your transactions in the dApp with Metamask. It also serves as a wallet for ETH and `ERC20 tokens`, such as DAI, TIN or DROP and allows you to transfer and receive such tokens.
+
+### How to install Metamask in your browser
+1. Go to https://metamask.io/download.html.
+2. Choose the browser you want to use.
+3. You will be directed to corresponding web store.
+4. Follow the steps there to add the extension to your browser.
+5. After setting up your wallet, Metamask should pop-up as a Fox icon ![](https://storage.googleapis.com/centrifuge-hackmd/upload_7dd07df8a7fc7d1b29743ec9535dbba7.png =20x) in your browser bar.
+
+### How to use Metamask with Tinlake
+- Connect your account.
```

Comment:
How about removing this bullet and turning it into the heading above? I.e.
### Connect MetaMask with Tinlake